PaxLabs/mynewt-core
# PKF-691: Enable low current mode by set LC_EN during initialization

Test:
1. have a unit can bypass battery and connect to battery emulator
2. Check the standby current. 
3. Run "power sleep". The current should go down to couple hundred uA.
